client/src/util/helpers.ts

```diff
@@ -188,4 +227,12 @@ export default {
     });
     return arr.join(" ");
   },
+
+  getTranslationOfComplaintType(complaintType: string, i18n: I18n) {
+    const translatedType = hpdComplaintTypeTranslations.get(complaintType);
+    if (!translatedType) {
+      reportError(`The HPD Complaint type "${complaintType}" isn't internationalized`);
```

Comment:
@samaratrilling, I remember you bringing up a good point on _when_ we should check whether there is a new complaint type from HPD data that we don't have translated/internationalized—specifically, that we should throw this kind of error from our [nycdb-k8s-loader](https://github.com/JustFixNYC/nycdb-k8s-loader/) when we actually generate the `wow_bldgs` table every morning vs. waiting for the issue to emerge from a user interacting with the front end. I thought this solution could work for now, so I'll file an issue for your solution once this is merged.